primitives/runtime/src/generic/unchecked_extrinsic.rs

```diff
@@ -106,13 +106,20 @@ impl<Address, Call, Signature, Extra: SignedExtension> Extrinsic
 {
 	type Call = Call;
 
-	type SignaturePayload = (Address, Signature, Extra);
```

Comment:
ahh, it is being done already 🙈

Reply:
In this case, a `type SignaturePayloadOf<E> = (<E as Extrinsic>::X, <E as Extrinsic>::Y, <E as Extrinsic>::Z)` will help prevent potential error where the order of these 3 might be mistaken.